### PR TITLE
Metadata change

### PR DESCRIPTION
**Pull request by shashank-boyapally**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description
Creating metadata as a new field would allow us to keep the metric and test name seperate and not have checks in the code.. This will help in generalization of fmatch/orion if we want it in the future.

## Related Tickets & Documents

- Related Issue #16 
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: utils/orion_funcs.py [hunk 1]
```diff
@@ -111,10 +111,7 @@ def get_metadata(test,logger):
         dict: dictionary of the metadata
     """
     metadata = {}
```
Comment (paigerube14) [inline utils/orion_funcs.py:113]: think we can take this line out becuase its being immediately overwritten with the next line

**Comment by paigerube14** [comment]
Lets also update the readme files with this change

File: utils/orion_funcs.py (hunk 1 above)
Comment (shashank-boyapally) [reply utils/orion_funcs.py]: Yep, sorry I did not check that! updated in a new push. Thank you!

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Review by jtaleric — APPROVED** [review]
lgtm

File: examples/small-scale-node-density-cni.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    - name:  etcdCPU
+      metricName : containerCPU
+      labels.namespace.keyword: openshift-etcd
+      metric_of_interest: value
+      agg:
+        value: cpu
+        agg_type: avg
+    
+    - name:  etcdDisck
```
Comment (krishvoor) [inline examples/small-scale-node-density-cni.yaml:50]: Spell check `etcdDisk`

**Review by jtaleric — APPROVED** [review]
lgtm

**Review by krishvoor — APPROVED** [review]
/lgtm

**Pull request merged**